app/build.gradle

```diff
@@ -99,6 +99,8 @@ dependencies {
     // Declare the dependency for the Cloud Storage library
     // When using the BoM, you don't specify versions in Firebase library dependencies
     implementation 'com.google.firebase:firebase-storage-ktx'
+    implementation 'com.google.firebase:firebase-firestore-ktx:24.0.2'
+
```

Comment:
This isn't needed and might cause issues, it's already included in line 93.

Reply:
removed :) plz check!